rust/Cargo.lock.in

```diff
@@ -886,18 +886,18 @@ dependencies = [
 
 [[package]]
 name = "serde"
-version = "1.0.209"
+version = "1.0.197"
```

Comment:
As in the other PR, can be done in our `Cargo.toml.in` so they stick over other updates.

Reply:
Same commit here. In the other PR you said lock, not toml.
Which is it ?

Here I try to just revert some changes and to make this revert minimal, so that CI is green again as soon as possible.

For example, We could also use serde 203 which is compatible, but I go this way to be sure

Reply:
I believe it needs to be in both

Reply:
> Same commit here. In the other PR you said lock, not toml.

Lock was wrong, sorry. In Cargo.toml.in, then it will be saved during other others to the toml and not easily lost like a change that is only to the lock file which can easily be wiped out.

Reply:
We need both for correct vendoring, right?

Reply:
I'm lost but this may be relevant:
1. The patch I made after which fuzz job broke **only** changed `Cargo.lock.in`: https://github.com/OISF/suricata/commit/599a451e44131e8c52a74eb175fb143e7db2ef6b
2. `serde` is not there at all in our `Cargo.toml`

Reply:
My understanding is:
1. we specify dependencies in `Cargo.toml.in` only. Also for overriding transitive dependencies.
2. we regen `Cargo.lock.in` after toml updates

Reply:
Added a commit with `Cargo.toml.in` changes in next version